features/importing_products_from_queue.feature

```diff
@@ -45,3 +45,15 @@ Feature: Importing products from queue
     And there is one item to import with identifier "braided-hat-l" for the "Product" importer in the Akeneo queue
     When I import all items in queue
     Then there should not be any temporary file in the temporary files directory
+
```

Comment:
While we are working on this, we could extract a simple "Background" with the first two steps, so we make all Scenarios easier to read.